tests/test_config.py

```diff
@@ -0,0 +1,141 @@
+import sys
+import os
+import pathlib
+import pytest
+
+_rundir = pathlib.Path(__file__).parent
```

Comment:
Yeah, that's a good idea.  That ought to be in config rather than db, though, but wherever it is, there's going to be a chicken and egg problem, though your next comment is really the right way to do it.